### PR TITLE
cs: Query for ZipSlip vulnerability (CVE-2018-1002200)

### PR DESCRIPTION
**Pull request by denislevin**
Initial check in to validate the tests

### CODE REVIEW

**Comment by calumgrant** [comment]
I have created a pull request against your branch to fix the failing unit tests.
https://github.com/denislevin/ql/pull/1

File: csharp/ql/src/Security Features/CWE-022/ZipSlip.ql [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/**
+ * @name Potential ZipSlip vulnerability 
+ * @description When extracting files from an archive, don't add archive item's path to the target file system path. Archive path can be relative and can lead to 
+ * file system access outside of the expected file system target path, leading to malicious config changes and remote code execution via lay-and-wait technique 
+ * @kind problem
+ * @id cs/zipslip
+ * @problem.severity error
+ * @precision high
```
Comment (calumgrant) [inline csharp/ql/src/Security Features/CWE-022/ZipSlip.ql]: Could you remove this line containing the  `@precision` tag. We are introducing a new workflow whereby we will add the `@precision` tag later when the query is ready to be deployed by default on LGTM.

**Comment by denislevin** [comment]
Removed @Precision tag, merged unit test fixes from #1 branch

**Comment by lukecartey** [comment]
Thanks @denislevin. This is ready to merge apart from the CLA, which was updated this morning, so unfortunately you'll need to sign it again - sorry about that!

**Comment by denislevin** [comment]
Luke,
Is there any way to get the diff of new CLA and old CLA? I can’t find it in GitHub.
Thanks!
-Denis

**Pull request merged**